test/lit-mobx.test.ts

```diff
@@ -22,13 +22,20 @@ export class TestMobxLitElement extends MobxLitElement {
     @observable
     public observableArray?: string[];
 
+    @observable
+    public observableValue: number = 1;
+
     @computed
     private get arrayLength(): string {
         return `Array length = ${
             this.observableArray ? this.observableArray.length : 'N/A'
         }`;
     }
 
+    public firstUpdated() {
```

Comment:
I would test mutations in `firstUpdated`, `updated`, and in `update` after the super call.

Reply:
Done in eb46001